Makefile

```diff
@@ -150,6 +150,9 @@ release: VERSION = v$(MAJOR).$(MINOR).$(PATCH)
 release:
 	scripts/release.sh $(VERSION)
 
+release-tag:
```

Comment:
As per comment above, I would recommend renaming this to `version-tag`

Reply:
That was not required anymore. Removed.